box2dbody.cpp

```diff
@@ -233,13 +232,21 @@ void Box2DBody::append_fixture(QQmlListProperty<Box2DFixture> *list,
     body->mFixtures.append(fixture);
 }
 
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
```

Comment:
I'm not sure if it is cleaner but instead of having a ifdef on the version, we can use `QList<Box2DFixture>::size_type` which work on both versions

Reply:
I agree, that is indeed cleaner.